App/Views/ContentView.swift

```diff
@@ -42,7 +43,14 @@ struct ContentView: View {
                 if newPhase == .background, store.state.preventScreenLock {
                     store.dispatch(.setPreventScreenLock(enabled: false))
                 }
+                
+                if newPhase == .active {
```

Comment:
This Shouldn't hit the widget refresh budget @creepymonster 
<img width="949" alt="Screen Shot 2022-11-25 at 10 56 26 AM" src="https://user-images.githubusercontent.com/34410446/204029454-d1d12da0-8d59-4167-838c-ec654837e4c1.png">

https://developer.apple.com/documentation/widgetkit/keeping-a-widget-up-to-date

Reply:
Thanks for the info. That means you could also update the widget whenever the app comes to the foreground.

I have always given the widgets (on the HomeScreen) a wide berth, because the update rate was not reliable enough for me. On the lockscreen I could get over it, but when the phone is unlocked anyway, the display on the app icon seemed more reliable (and not further away).